examples/atari/atari_wrapper.py

```diff
@@ -181,6 +181,20 @@ def reward(self, reward):
         return np.sign(reward)
 
 
+class NoRewardEnv(gym.RewardWrapper):
+    """sets the reward to 0.
+
+    :param gym.Env env: the environment to wrap.
+    """
+
+    def __init__(self, env):
+        super().__init__(env)
+
+    def reward(self, reward):
+        """Set reward to 0."""
+        return np.zeros_like(reward)
+
+
```

Comment:
Do we really need it? I don't see any call.

Reply:
It's called in `irl_gail.py`:

```python
from examples.atari.atari_wrapper import NoRewardEnv
...
    train_envs = SubprocVectorEnv(
        [lambda: NoRewardEnv(gym.make(args.task)) for _ in range(args.training_num)],
        norm_obs=True
    )
```

This is to make sure the implementation doesn't accidentally use the reward signal coming from the env. I like that it gives me more confidence about the correctness.

Reply:
But you apply an Atari wrapper for Mujoco task? I mean it's not the right place to put this wrapper.

Reply:
I moved it to `irl_gail.py`.